scripts/jenkins/Makefile.jenkins

```diff
@@ -38,8 +38,24 @@ test-package-r:
 		$(JENKINS_MAKEFILE_PATH) tests/runitSmokeTestList tests/runitAutoMLList \
 		scripts/validate_r_cmd_check_output.R scripts/validate_r_cmd_check_output.py h2o-3-DESCRIPTION
 
+init-tests:
+	>.tests-r.txt
```

Comment:
why do we need to append to the tests? would it be better to just write the test file once? without appending?

Reply:
this makes combining multiple sets of exclusions (or potentially inclusions) possible - for example, see targets `test-r-small` or `test-r-small-client-mode` which have multiple lookup dependencies

Reply:
HDFS testes are not run as a part of test-r-small or test-r-small-client-mode.

Reply:
sure, that's why they are excluded - the lookup is universal (for both include and exclude) and then the list is used with parameter, either `--testlist` or `--excludelist`

Reply:
I know it's not ideal, but should be simpler than before... I will add some brief explanatory comments